supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
+* Do not use non hyperlinks.
```

Comment:
What does "non hyperlinks" mean? Does it mean that you provide a URL in the text, but it's not clickable?

Reply:
I don't know sorry, maybe we should ask JP because this is form his best practices doc?

Reply:
It took me a while to figure out what I meant when I wrote "non hyperlinks" - and i understand it's confusing - however, it means that there are many types of links (Java, Flash, Image... and a few more) my intent is to show that for links in HTML, only use HREF links and not any other type of links. Here's a good article that clarifies it [Types of Links: DeepCrawl](https://www.deepcrawl.com/knowledge/technical-seo-library/different-types-of-links/) - (we use DeepCrawl as our site crawling service)